doc/source/tutorial-quickstart-scikitlearn.rst

```diff
@@ -65,34 +60,20 @@ The pre-defined functions are used in the :code:`client.py` and imported. The :c
   from flwr_datasets import FederatedDataset
 
 Prior to local training, we need to load the MNIST dataset, a popular image classification dataset of handwritten digits for machine learning, and partition the dataset for FL. This can be conveniently achieved using `Flower Datasets <https://flower.ai/docs/datasets>`_.
-The :code:`FederatedDataset.load_partition()` method loads the partitioned training set for each partition ID defined in the :code:`--partition-id` argument.
+The :code:`FederatedDataset.load_partition()` method loads the partitioned training set for each partition ID set in the `partition_id` variable. We assign an integer to each `partition_id` for each client in our federated learning example, starting from 0.
```

Comment:
There is no more `partition_id`, as far as I know it is replaced with `load_partition(node_id=node_id)`

Reply:
@danielnugraha Hmm, I checked our code and I can only find `partition_id` - see for example in [this line](https://github.com/adap/flower/blob/4dc80062c5b810de78279a88f94884b460a38fee/datasets/flwr_datasets/partitioner/partitioner.py#L56). Did I understand this correctly? cc @adam-narozniak?

Reply:
Closing old thread as it's no longer relevant.